### ISSUE TITLE
[BUG] in GET /v2/types do not count the total of untyped entities and the type of attributes are not returned

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1636]
In GET  `/v2/types` 

##### 1. it is not shown the total of untyped entities.

Example:
```
{
    "":{
        "attrs":{
            "pepe":{
                "type":""
            },
            "timestamp":{
                "type":""
            }
        },
        "count":0
    }
}
```
Also the `options=count` query parameter does not respond the entities total. Entities untyped are not added in ` x-total-count` response header.

---------------------------------------------

#####  2. The attribute type are not returned if the attribute name match with another in the same group.  https://gist.github.com/iariasleon/ef86546a97d757a11568 

*Scenario*:
If two entities have the same "type" (but with different "id"), with the same attribute name too, but with different attribute type (or one with attribute type and another it is missing), both are shown with only an attribute type. 

*Expected*:
    - two attributes names with its attribute type respective
    - one attribute name but with two attributes types

-------------------------------------------
General Info for both: https://gist.github.com/iariasleon/8dc197e909d4fb72e45d

**Comment by fgalan** [comment]
This issue is pending on some internal discussion with @jmcanterafonseca , so we defer it to 0.28.0.

**Comment by fgalan** [comment]
Fix this issue *after* implementing https://github.com/telefonicaid/fiware-orion/issues/1833

**Comment by iariasleon** [comment]
LGTM about count the total of untyped entities.

The another theme **The attribute type are not returned if the attribute name match with another in the same group** I think is better, move it at another new issue, What do you think?

**Comment by fgalan** [comment]
Fixed in PR https://github.com/telefonicaid/fiware-orion/pull/2010. Please @iariasleon have a look and close the issue in that case.

**Comment by iariasleon** [comment]
This issue will be re-tested when "/v2/types" feature is implemented

**Comment by iariasleon** [comment]
LGTM
